bin/bundle

```diff
@@ -1,3 +1,109 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
-load Gem.bin_path('bundler', 'bundle')
+# frozen_string_literal: true
+
+#
+# This file was generated by Bundler.
+#
+# The application 'bundle' is installed as part of a gem, and
+# this file is here to facilitate running it.
+#
+
+require "rubygems"
```

Comment:
does Rubocop run on these files? I'd expect it to be upset at the double quotes here

Reply:
I don't think so. I can add it though.

Reply:
I don't feel that strongly about it, we can leave it out if it's out